src/WooCommerce.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * @file
+ * Contains \Netzstrategen\Gallerya\WooCommerce.
+ */
+
+namespace Netzstrategen\Gallerya;
+
+/**
+ * WooCommerce custom functionality.
+ */
+class WooCommerce {
+
+  /**
+   * Adds srcset attributes to image links to make them reponsive on lightgallery.
+   *
+   * @see https://sachinchoolur.github.io/lightGallery/demos/responsive.html
+   *
+   * @implements woocommerce_single_product_image_thumbnail_html
+   */
+  public static function woocommerce_single_product_image_thumbnail_html($html, $thumbnail_id) {
+    $srcset = wp_get_attachment_image_srcset($thumbnail_id, 'shop_single');
```

Comment:
Aren't we parsing the output of an image?  What if there is a `srcset` attribute on the image already?

Reply:
Please read https://sachinchoolur.github.io/lightGallery/demos/responsive.html.

The only way is to pass `data-srcset`, `data-size` or `data-responsive` to the wrapping `a` element. So, if there is already a `srcset` applied to the image, is not helpful.